sup.rb

```diff
@@ -0,0 +1,41 @@
+require "language/go"
+
+class Sup < Formula
+  desc "Super simple deployment tool - just Unix - think of it like 'make' for a network of servers."
+  homepage "https://github.com/pressly/sup"
+  url "https://github.com/pressly/sup/archive/4ee5083c8321340bc2a6410f24d8a760f7ad3847.zip"
```

Comment:
That's would be great. 

I think let's call it 0.4 from current master. because I can see you guys add some thing/bug fix since 0.3 on Dec 15th.